core/services/versionchooser/static/style.css

```diff
@@ -0,0 +1,23 @@
+body {
+  background-color: #ECF0F5;
+}
+
+.panel .panel-title .btn-custom {
+width: auto;
+}
+
+.disabled {
+cursor:not-allowed;
+}
+
+#repo {
+width: 50%;
+}
+
+.docker-pull-output {
+  white-space: pre; font-family:monospace;
+}
+
+.image-small {
+  font-size: 70%;
+}
```

Comment:
Is eslint checking the CSS formatting? Seems that it is ignoring the indentation here.

Reply:
right, I made two of the endpoints sync and fixed the css. I opened #191 for the css validation.